### ISSUE TITLE
user must wait for playback start

### ISSUE THREAD

**Issue by 532910** [y20k/transistor#320]
Steps:
1. Press play button
2. Switch to desktop or another app or close transistor

Expected behavior: playback must start

**Comment by y20k** [comment]
Interesting bug. I am not able to reproduce, yet. I have a station that takes quite a while to start (~2s). When I tap start and immediately go to the home screen the station still starts playback in background.

**Comment by 532910** [comment]
> not able to reproduce

Hm, it's strange. It's reproducible for me 100% of tries. Could you try with mobile or some slow connection, far away from wifi AP for example.

https://sergio.outerface.net/misc/ScreenRecord-2020-12-07-20-30-46.mp4

**Comment by y20k** [comment]
Yeah. I'll keep trying. Pretty trippy home screen background, btw.

**Comment by 532910** [comment]
Haa, thank you! It is [Blurred Lines](https://f-droid.org/packages/cxa.lineswallpaper), and you haven't seen PopSqures from XScreenSaver on my desktop yet (:
